layers/+lang/ruby/funcs.el

```diff
@@ -160,9 +225,5 @@ Called interactively it prompts for a directory."
 
 ;; Prettier
 
-(defun spacemacs/ruby-format ()
-  (interactive)
-  (call-interactively 'prettier-js))
-
-(defun spacemacs/ruby-fmt-before-save-hook ()
-  (add-hook 'before-save-hook 'spacemacs/ruby-format t t))
+(defun spacemacs//ruby-add-prettier-js-before-save-hook ()
+  (add-hook 'before-save-hook 'prettier-js t t))
```

Comment:
the third argument to `add-hook` is `DEPTH`.
It defaults to 0, and any non-nil symbol implies 90.
So this hook is likely to be run after other hooks (-100 is first and 100 is last).

Is this really the behaviour we want?

Reply:
I guess so. The addition of `DEPTH` is a rather new one and it is meant to be backward compatible. 
If there was `t` to append, then `DEPTH` should keep the same behavior.